srv/GetNamedPoses.srv

```diff
@@ -0,0 +1,8 @@
+# Get predefined poses which are convenient to for spawning, navigation goals etc.
+# This is an optional simulation feature: check status_message if you receive an empty poses vector,
+# to determine whether this feature is not supported by your simulator, or named poses are simply not defined.
+
```

Comment:
Named poses have tags. What about adding a filter here, too?

Reply:
Good catch, adding for consistency, and also separating out a new message (TagsFilter)